Objects/dictobject.c

```diff
@@ -3108,10 +3123,17 @@ _PyDict_MergeEx(PyObject *a, PyObject *b, int override)
     return dict_merge(interp, a, b, override);
 }
 
+/*[clinic input]
+dict.copy
+
+D.copy() -> a shallow copy of D
```

Comment:
```suggestion
Returns a shallow copy of the dictionary.
```

Reply:
We use imperative mood for the reference; I added suggestions to amend the docstrings.